policyeval/base_worker.go

```diff
@@ -86,6 +87,7 @@ func (w *BaseWorker) Run(ctx context.Context) {
 		// Notify broker that policy eval was successful.
 		if err := w.broker.Ack(eval.ID, token); err != nil {
 			logger.Warn("failed to ACK policy evaluation", "err", err)
+			os.Exit(1)
```

Comment:
Is this a change we are expecting to include? If so, could you provide a little context on the change and consider moving it to a separate PR?

Reply:
Ahh woops, this snuck in! -- I was testing this weeks ago to restart the autoscaler when the evaluation threads would hang.  My bad, removing that.